OrbitCore/CaptureData.cc

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "CaptureData.h"
```

Comment:
probably you wanted .cpp here.

Reply:
or just remove this file.

Reply:
I thought we were moving to cc files to meet google style guide, aren't we? Anyways, I can rename it for sure. However, I'd like to already include this, as there is more to come :D

Reply:
if we do - we should probably do it in bulk

Reply:
okay, I changed the file ending